cri/v1alpha2/cri_utils.go

```diff
@@ -205,14 +205,20 @@ func applySandboxSecurityContext(lc *runtime.LinuxPodSandboxConfig, config *apit
 		sc = &runtime.LinuxContainerSecurityContext{
 			SupplementalGroups: lc.SecurityContext.SupplementalGroups,
 			RunAsUser:          lc.SecurityContext.RunAsUser,
+			RunAsGroup:         lc.SecurityContext.RunAsGroup,
+			RunAsUsername:      "",
```

Comment:
If RunAsUsername is "", don't need to initialize it.  Its default value is "".

Reply:
Updated. ^_^